microsoft-edge/devtools-guide-chromium/whats-new/whats-new-archive.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What's new in Microsoft Edge DevTools Archives
+description: New features and fixes in Microsoft Edge DevTools Archives
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/23/2023
+---
+# Archived What's New entries
+
+This page is an archive for all of the legacy [What's New in DevTools](whats-new.md) entries published after each release.
```

Comment:
```suggestion
The following What's New entries cover older releases.
```
Prefer describing the technology instead of describing the docs.
Draft wording alternatives:
The following features were introduced in earlier releases.

_____________
A separate comment:
Prefer link pattern using one of these options:

For more info see [link]

See also:
* [link]

h2 See also
* [link]